ArduCopter/Parameters.cpp

```diff
@@ -30,6 +30,12 @@
 #endif
 
 const AP_Param::Info Copter::var_info[] = {
+
+     // @Param: ACL_LIMIT
+    // @DisplayName: Eeprom format version number
+    // @Description: Limit acceleration
+    // @User: Advanced
+    GSCALAR(adjusted_acl, "ACL_LIMIT",   100),
```

Comment:
I think the purpose of this change is to allow users to more aggressively move the vehicle around while Landing.  So this is related to the LAND_REPOSITION parameter/feature.

So we should probably rename this parameter to start with LAND_.  Perhaps "LAND_ACCEL" would be good and default it to zero meaning that WPNAV_ACCEL is used unless this parameter is set.

The parameter descriptions need some improvment as well.  "Eeprom format version number" looks like a copy-paste error.  We should also specify the units and those should be m/s/s.